packages/authentication/modules/access/useRecoverPassword/__tests__/useRecoverPassword.test.tsx

```
@@ -1,28 +1,18 @@
-  mockFetchError,
-  renderHook,
-} from '@baseapp-frontend/test'
+import { ComponentWithProviders, MockAdapter, renderHook } from '@baseapp-frontend/test'
+import { axios } from '@baseapp-frontend/utils'
 
 import { z } from 'zod'
 
 import useRecoverPassword from '../index'
 
+// @ts-ignore TODO: (BA-1081) investigate AxiosRequestHeaders error
+export const axiosMock = new MockAdapter(axios)
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)